apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+---
+eleventyNavigation:
+    key: Code
+    parent: Iconography
+    order: 3
+---
+
+## Overview
+- [Our packages](#our-packages)
+- [Pie Icons](#pie-icons)
+  - [Usage - Client-side Javascript](#usage-client-side-javascript)
+    - [Installation](#installation)
+    - [Include the package](#include-the-package)
+    - [HTML](#html)
+    - [Invoke the package](#invoke-the-package)
+  - [Usage - NodeJS](#usage-nodejs)
+    - [Installation](#installation-1)
+    - [Require the package](#require-the-package)
+    - [Call the APIs](#call-the-apis)
+- [React](#react)
+  - [Usage](#usage)
+    - [Installation](#installation-2)
+    - [Import into your project](#import-into-your-project)
+- [Vue](#vue)
+  - [Usage](#usage-1)
+    - [Installation](#installation-3)
+    - [Import into your project](#import-into-your-project-1)
+- [Web Components](#web-components)
+  - [Usage - In Lit Components](#usage-in-lit-components)
+    - [Installation](#installation-4)
+    - [Import into your project](#import-into-your-project-2)
+
+
+---
+
+## Our packages
+
+We have a number of different icon packages available for use in our applications. We have a package for each framework we support, as well as a base package for the icons themselves.
+
+---
+
+## Pie Icons
+
+`@justeattakeaway/pie-icons` is our base icon package. It is a collection of SVG files. This means that you can use these icons in all the same ways you can use SVGs (e.g. img, background-image, inline, object, embed, iframe).
+
+[This package](https://github.com/justeattakeaway/pie/tree/main/packages/tools/pie-icons) is used to build the framework-specific icons for Web Components, React and Vue.
+
+
+{% notification {
+  type: "warning",
+  message: "Whilst this package can be used for web applications, we strongly advise that you use a framework-specific implementation instead."
+} %}
+
+### Usage - Client-side Javascript
+
+To use the icons in your application, you need to include the package in your project, add the icons using data-attributes and invoke the `replace()` function:
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons
+```
+
+#### Include the package
+You can load directly from the installed package, or from a CDN provider.
+
+```html
+<script src="path/to/dist/pie-icons.js"></script>
+```
+```html
+<!-- choose one -->
+<script src="https://unpkg.com/@justeattakeaway/pie-icons"></script>
+<script src="https://cdn.jsdelivr.net/npm/@justeattakeaway/pie-icons/dist/pie-icons.min.js"></script>
+```
+
+#### HTML
+Simply add a `data-pie-icons` attribute with the icon name to an element. You can see all of our icons on the [library page](http://localhost:8080/foundations/iconography/library/).
+
+```html
+<i data-pie-icons="menu"></i>
```

Comment:
Is `i` the right element here?

Reply:
It's plucked from the package readme, the entire element is replaced so I don't think it hugely matters but as I understand historically some icon libraries have used the `i` element for icon placeholders which is why I think this package suggests the same.